test/e2e/certificates_test.go

```diff
@@ -171,10 +185,109 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate on ibm cloud CIS", func() {
+			cisCRN, isCisCRN := os.LookupEnv(cisCRNEnvironmentVar)
+			if targetPlatform, ok := os.LookupEnv(targetPlatformEnvironmentVar); ok && targetPlatform == "ibmcloud" {
```

Comment:
Alternatively, could we resort to checking the `Infrastructure` object instead? I'm guessing that checking `infra.Status.PlatformStatus.IBMCloudPlatformStatus` is a much better option.

Reply:
We'll double check this, however, I don't think this is going to work with the UPI install we use in IBM Cloud PowerVS.

Reply:
As a follow-up... 
```oc get infrastructure cluster -ojson | jq -r '.spec.platformSpec.type'
None
```
We cannot use this as it reports None. It is a UPI installation.

Reply:
If you'd like we can add this to a ConfigMap that is manually setup.

Reply:
Thanks for the clarity on this, could you please help document here itself that the 
IBM CIS CRN is something that you inject from our side or if there are scripts/controller already running on your UPI cluster to do that for us.

Reply:
Hey @pranitaT can you add that please